src/framework/dockwindow/thirdparty/KDDockWidgets/src/private/quick/FrameQuick_p.h

```diff
@@ -71,7 +71,7 @@ class DOCKS_EXPORT FrameQuick : public Frame
     void tabTitlesChanged();
 
 public Q_SLOTS:
-    void updateConstriants();
+    void updateConstraints();
```

Comment:
This is actually third party code, so usually I'd avoid making changes to it unless strictly necessary. But we've already made quite a mess of this library anyway, so you can keep the changes for now.